install/installer/pkg/components/cluster/podsecuritypolicies.go

```diff
@@ -100,7 +101,7 @@ func podsecuritypolicies(ctx *common.RenderContext) ([]runtime.Object, error) {
 				},
 				HostNetwork: true,
 				HostPorts: []v1beta1.HostPortRange{{
-					Min: 30000,
+					Min: 20000,
```

Comment:
Hm. But this would add all ports from 20000-33000 range into SecPolicy. 
While registry facade is only using port 20000?

Reply:
fixed